websites/C/Coub/dist/metadata.json

```diff
@@ -40,6 +40,18 @@
 			"title": "Show current playing button",
 			"icon": "fas fa-external-link-alt",
 			"value": true
+		},
+		{
+			"id": "show_externalLargeImage",
+			"title": "Show community/channel images",
+			"icon": "fas fa-image",
+			"value": true
+		},
+		{
+			"id": "show_startedBrowsing",
+			"title": "Show elapsed time",
+			"icon": "fas fa-stopwatch",
+			"value": true
 		}
```

Comment:
What would be the reason why you wouldn't want this? Haven't seen many presences give this as a setting

Reply:
someone requested to show elapsed time, i didnt liked it so i added a toggle

Reply:
Ok, use Title Case on the setting titles